examples/call/commands/connected/call.ts

```diff
@@ -0,0 +1,31 @@
+import { ethers } from "ethers";
+
+import { createCommand, createRevertOptions, getAbi } from "../common";
+
+const main = async (options: any) => {
```

Comment:
Let's ensure we don't use `any`, these commands would benefit from zod argument validation, this function is an excellent example of why this is necessary because it uses `types` and `values`, and since we're not using any validation or refinements, they could differ in length. We could use a commander preAction hook if we think zod is overkill for this small project.

Reply:
Examples is a different sort of code base where we can't have extra code laying around. I agree with you about additional validation and I think we can accomplish this by exporting required validation rules/functions from the toolkit and just importing them here.